fix_includes.py

```diff
@@ -1944,7 +2017,9 @@ def _GetSymbolNameFromForwardDeclareLine(line):
   """
   iwyu_namespace_re = re.compile(r'namespace ([^{]*) { ')
   symbolname_re = re.compile(r'([A-Za-z0-9_]+)')
-  namespaces_in_line = iwyu_namespace_re.findall(line)
+  # turn anonymous namespaces into their proper symbol representation
```

Comment:
Google style is proper sentence casing and punctuation for comments, so "Turn ... symbol representation." with a closing period.